extensions/wikia/Search/classes/Query/Select.php

```diff
@@ -69,7 +69,7 @@ public function __construct( $queryString ) {
 	 */
 	public function getSanitizedQuery() {
 		if ( $this->sanitizedQuery == null ) {
-			$this->sanitizedQuery = html_entity_decode( (new Sanitizer)->StripAllTags( $this->rawQuery ), \ENT_COMPAT, 'UTF-8');
+			$this->sanitizedQuery = html_entity_decode( Sanitizer::stripAllTags( $this->rawQuery ), \ENT_COMPAT, 'UTF-8');
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "UTF-8" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)